app/client/components/productPage.tsx

```diff
@@ -167,123 +168,167 @@ const getPaymentPart = (
 };
 
 const getProductRenderer = (productType: ProductTypeWithProductPage) => (
-  apiResponse: MembersDataApiResponse
+  apiResponse: MembersDataApiResponse[]
 ) => {
-  if (hasProduct(apiResponse)) {
-    const data: ProductDetail = apiResponse;
-    if (data.subscription.cancelledAt) {
-      return getCancellationSummary(productType)(data.subscription);
-    }
-    return (
-      <div>
-        {data.alertText ? (
+  const productDetailList = apiResponse.filter(hasProduct);
+  return (
+    <>
+      {productDetailList.length > 1 ? (
+        <PageContainer>
+          <h3>
+            It appears you have {productDetailList.length} concurrent{" "}
+            {productType.friendlyName}s...
+          </h3>
+        </PageContainer>
+      ) : (
+        undefined
+      )}
+      {productDetailList.length > 0 ? (
+        productDetailList.map((productDetail, listIndex) => (
           <div
+            key={productDetail.subscription.subscriberId}
             css={{
-              backgroundColor: palette.red.dark,
-              color: palette.white,
-              padding: "10px 15px 15px",
-              marginTop: "30px",
-              marginBottom: "30px"
+              background:
+                productDetailList.length > 1 && (listIndex + 1) % 2 !== 0
```

Comment:
i wanted the first entry to be grey so that the intro line above it can remain with no background (white). However in the scenario where there is only one (where it displays as it used to before) we want that with no background.